tools/ldb_cmd.cc

```diff
@@ -100,17 +101,24 @@ const std::string LDBCommand::ARG_BLOB_GARBAGE_COLLECTION_FORCE_THRESHOLD =
 const std::string LDBCommand::ARG_BLOB_COMPACTION_READAHEAD_SIZE =
     "blob_compaction_readahead_size";
 const std::string LDBCommand::ARG_DECODE_BLOB_INDEX = "decode_blob_index";
+const std::string LDBCommand::ARG_DUMP_UNCOMPRESSED_BLOBS =
+    "dump_uncompressed_blobs";
 
 const char* LDBCommand::DELIM = " ==> ";
 
 namespace {
 
+using ROCKSDB_NAMESPACE::blob_db::BlobDumpTool;
```

Comment:
We could move this into `DumpBlobFile` since that's the only place it's used I believe.